charts/kubearchive/templates/database/ps-claim.yaml

```diff
@@ -4,13 +4,13 @@ apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
   labels:
-    app: {{ .Values.database.name }}
-  name: {{ .Values.database.name | default "postgresql" }}-volume-claim
-  namespace: {{ .Values.kubearchive.namespace }}
+    app: {{ tpl .Values.database.name . }}
+  name: {{ tpl .Values.database.name . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
I think we decided to remove the namespaces. Also, this line is redundant with the default behaviour. If no namespace is defined, the namespace is the defined by the release.

Reply:
+1 to the line being redundant. One of the points defined in the naming conventions is that we will remove the namespace of the resource when they are "kubearchive" as we will be providing it from the `-n` option on the `helm install` command.

But I don't think we agreed on removing the spaces. I think you might be referring to the issue https://github.com/kubearchive/kubearchive/issues/67, but as Sam mentions, `{{ variable }}` is the preferred Helm syntax. Also, note that this won't be a problem from the YAML linter POV as the `yamllint` will be applied after the `helm template` command render the values on the `charts/generated/` directory.

Reply:
removed the namespace`